### ISSUE TITLE
Enhanced ReFramework Documentation Layer

### ISSUE THREAD

**Issue by xiboyuk** [mihhdu/UiPath_REFramework#4]
Hi,
Your documentation is not so different from classic ReFramework's Documentation. It doesn't have any information about some xaml files such as "CleanupAndPrep.xaml" and "GetSetTransactionData.xaml"

Will you gonna update this framework and will it be ever implemented in UiPath officially? What is the latest state? And how we can learn more information about processes. It is actually complicated because lots of "xamls" are the same. It confuses about logic. 

In which processes we should use classic reframework or enhanced one? We thought this should have sub reframework parts but design is not like that. What is the logic behind folder structure like service layer or process layer? Why we are invoking "main.xaml" and process layer in every layer?
Thank you.

**Comment by xbsuersan** [comment]
@mihhdu 
Do you have any example processes with task part usage?

**Comment by erashishgupta84** [comment]
Yes. I do have question regarding invoking "main.xaml" and process layer in every layer. Please provide clarifications.
